pkgs/by-name/hy/hyprland/package.nix

```diff
@@ -75,6 +75,8 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-deu8zvgseDg2gQEnZiCda4TrbA6pleE9iItoZlsoMtE=";
   };
 
+  patches = [ ./asahi-fix.patch ];
```

Comment:
Add a comment on why the patch is necessary, also link the upstream PR/commit.

Amend your commit and force push.

Reply:
The commit can be `fetchpatch`ed, so the patch file itself can be removed.